browser/bundles/o-errors/filter-error.js

```diff
@@ -7,14 +7,16 @@ module.exports = ({ exception }) => {
 		/'undefined' is|undefined is|is undefined/i
 	];
 	let windowFtError;
+	let sourcepointError;
 	if(exception) {
 		try {
 			let errorString = JSON.stringify(exception);
 			windowFtError = errorFilters.every(rx => rx.test(errorString));
+			sourcepointError = /sourcepoint/.test(errorString);
```

Comment:
Why can't this go in the `errorFilters` array?

Reply:
Because I need to merge the other two values in there together into one regex and also because I suck at regex